packages/config/config/devices/0x0460/qnsw-001P16.json

```diff
@@ -73,102 +99,220 @@
+				{
+					"label": "close relay",
+					"value": 2
+				}
+			]
 		},
 		{
 			"#": "92",
 			"label": "Alarm Conf. - Smoke",
-			"description": "0",
+			"description": "This parameter determines which alarm frames the Device should respond to and how. The parameters consist of 4 bytes, the three most significant bytes are set according to the official Z-Wave protocol specification.0",
 			"valueSize": 4,
 			"minValue": 0,
 			"maxValue": 2,
 			"defaultValue": 0,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "no action",
+					"value": 0
+				},
+				{
+					"label": "open relay",
+					"value": 1
+				},
+				{
+					"label": "close relay",
+					"value": 2
+				}
+			]
 		},
 		{
 			"#": "93",
 			"label": "Alarm Conf. - Co",
-			"description": "0",
+			"description": "This parameter determines which alarm frames the Device should respond to and how. The parameters consist of 4 bytes, the three most significant bytes are set according to the official Z-Wave protocol specification.",
 			"valueSize": 4,
 			"minValue": 0,
 			"maxValue": 2,
 			"defaultValue": 0,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "no action",
+					"value": 0
+				},
+				{
+					"label": "open relay",
+					"value": 1
+				},
+				{
+					"label": "close relay",
+					"value": 2
+				}
+			]
 		},
 		{
 			"#": "94",
 			"label": "Alarm Conf. - Heat",
-			"description": "0",
+			"description": "This parameter determines which alarm frames the Device should respond to and how. The parameters consist of 4 bytes, the three most significant bytes are set according to the official Z-Wave protocol specification.",
 			"valueSize": 4,
 			"minValue": 0,
 			"maxValue": 2,
 			"defaultValue": 0,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "no action",
+					"value": 0
+				},
+				{
+					"label": "open relay",
+					"value": 1
+				},
+				{
+					"label": "close relay",
+					"value": 2
+				}
+			]
 		},
 		{
 			"#": "120",
 			"label": "Factory Reset",
-			"description": "0",
+			"description": "Reset to factory default settings and removed from the Z-Wave network.\nThe parameter is Advanced and may be hidden under the Advanced tag.",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 0,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "Don’t do Factory reset",
+					"value": 0
+				},
+				{
+					"label": "Do the Factory reset",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "201",
 			"label": "Serial Number 1",
-			"description": "0",
+			"description": "This parameter contains a part of device’s serial number.\nThe parameter is Read-Only and cannot be changed.\nThe parameter is Advanced and may be hidden under the Advanced tag.",
```

Comment:
I'd remove the description. That it is read-only is already included in the definition. And the last sentence doesn't give any useful information to the end user.

Reply:
same for the next ones